### DISCUSSION TITLE
Suggestion: Instructions with images

### DISCUSSION THREAD

**Discussion by droid51** [craftcms/cms#8448] [Ideas]
Hi there,

instructions with images would be cool (if you want to show the section of a page, an error-message, ...)
"One Picture is Worth Ten Thousand Words" :-) 
Thanks!

example image could be:
![grafik](https://user-images.githubusercontent.com/8912561/28576677-c6e5d394-7154-11e7-9d95-48818ac422bb.png)

Greetings
crazyx13th

**Comment by angrybrad** [comment]
Not sure I'm following... are you making a suggestion for the front-end of a Craft website?  If so, the front-end content in Craft is 100% up to you.  If you're making a suggestion for a more general Craft improvement, can you edit the original question to provide some more clarity?

**Comment by droid51** [comment]
no :-D I'm talking about the CP... and the image you see is an example of an inserted image


somthing like this, or an mouse over layer on an icon, or....
![grafik](https://user-images.githubusercontent.com/8912561/28610475-bfaf26ae-71e8-11e7-95f3-1a33c211df3a.png)


thx

**Comment by brandonkelly** [comment]
Did a test, and it actually _is_ possible to include images using Markdown’s image syntax:

    ![alt text](https://user-images.githubusercontent.com/8912561/28576677-c6e5d394-7154-11e7-9d95-48818ac422bb.png)

There’s some room for improvement – at least Craft should be limiting its max-width to 100%.

**Comment by droid51** [comment]
ahh ok, cool :-)
But with Markdown I had to upload an image at my own, right? Would be cool to uploded the images via "paste-clipboard-method"
thx!

**Comment by brandonkelly** [comment]
Correct, you would need to upload it first. I could see us swapping out the current Instructions setting with CKEditor 5 set to Markdown mode, once that's a thing (#1811), and that would have built-in image uploading capability.

**Comment by droid51** [comment]
sounds awesome! 👍
